datavault-webapp/src/main/java/org/datavaultplatform/webapp/controllers/admin/AdminRoleAssignmentController.java

```diff
@@ -0,0 +1,136 @@
+package org.datavaultplatform.webapp.controllers.admin;
+
+import org.datavaultplatform.common.model.RoleAssignment;
+import org.datavaultplatform.common.model.RoleModel;
+import org.datavaultplatform.common.model.RoleType;
+import org.datavaultplatform.common.model.User;
+import org.datavaultplatform.webapp.exception.EntityNotFoundException;
+import org.datavaultplatform.webapp.services.RestService;
+import org.hibernate.validator.constraints.NotEmpty;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+
+@Controller
+public class AdminRoleAssignmentController {
```

Comment:
Had thought about that, only reason I didn't was because doing anything with the schools required admin permissions, but that's not the case with vaults. Is that still the same case with schools?